src/io/flutter/utils/DiagnosticLevel.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2017 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.utils;
+
+/**
+ * The various priority levels used to filter which diagnostics are shown and
+ * omitted.
+ *
+ * Trees of Flutter diagnostics can be very large so filtering the diagnostics
+ * shown matters. Typically filtering to only show diagnostics with at least
+ * level debug is appropriate.
+ */
+public enum DiagnosticLevel {
```

Comment:
This is related to APIs exposed in the flutter framework? Perhaps reference the code / classes here? We may also want the javadoc to be terser to reduce our maintenance burden, esp. if these classes are well documented in the flutter framework.

Reply:
I copied and pasted the docs from the Flutter as you guessed. Added a link to the Flutter file with the API.

Reply:
kk. going forward, we may want to tersify the docs (or eliminate them?) to reduce drift and maintenence

Reply:
I don't expect these particular classes will change much in the future. If there are changes likely it will just be adding extra logging levels in methods but I could be wrong. Hopefully these pretty carefully written docs will be helpful for other people contributing to this code on the IntelliJ. If not, they should be slimmed down or removed.

Reply:
gotcha